CFC_DataCollector/recommender/recommendation_pipeline.py

```diff
@@ -0,0 +1,15 @@
+def get_trips_to_improve(user_id, filter_function):
```

Comment:
How is this different from improve_trips_pipeline?
We should retain only one of them.